AzureMonitorAgent/agent.py

```diff
@@ -1717,6 +1720,22 @@ def get_ssl_cert_info(operation):
 
     log_and_exit(operation, GenericErrorCode, 'Unable to determine values for SSL_CERT_DIR or SSL_CERT_FILE')
 
+def copy_kqlextension_binaries():
+    kqlextension_bin_local_path = os.getcwd() + "/KqlExtensionBin/"
```

Comment:
where are we adding contents to KqlExtensionBin? is that a separate PR?

Reply:
yes, that is separate PR in tux repo to populate the content of the folder. This PR is for installing the folder content as part of extension install.

Reply:
PR to populate kql bin: https://msazure.visualstudio.com/One/_git/Compute-Runtime-Tux/pullrequest/10283834